cli/src/commands/extensions/mod.rs

```diff
@@ -47,6 +47,8 @@ pub fn command<'a>() -> Command<'a> {
         .subcommand(
             Command::new("run")
                 .about("Run an extension from a directory")
+                .allow_hyphen_values(true)
+                .disable_help_flag(true)
```

Comment:
So that `--help` can be forwarded to the extensions that may want to print their own help.

Reply:
Roger that...understood. It might be surprising/confusing to have a `phylum` command without help, but an alternative solution is not coming to mind. The current documentation for this command ([in review](https://github.com/phylum-dev/cli/pull/562/files#diff-0f156a4fb7cba9f87e2a86038a1b13e9f6a8a8890c6955c95234ea9715ab5c5b)) says:

```sh
phylum extension run [OPTIONS] <PATH> [OPTIONS]...

[---TRIMMED---]

The first set of options are for the `run` command. The second set of options
are for the extension.
```

Is that statement about options false? It's not possible to have something like `phylum extension run --help` to print the `phylum` help message and also have something like `phylum extension run -y /path/to/ext --help` to print the extension's help message?

Reply:
I _think_ currently the `clap` crate does not have that capability, unless the `--help` comes after `--`. One thing we _could_ do, though quite a bit hacky, is manually matching the `PATH` parameter against `-h` / `--help` and print a help instead of running an extension.

Reply:
Actually you shouldn't disable the help flag on `run`, you should have it on the extension subcommands. And it should be on those already from my previous PR?

Reply:
I guess you could maybe add a custom help flag, but I feel like that would be more trouble than it's worth.